app/models/recipe.rb

```diff
@@ -1,3 +1,4 @@
 class Recipe < ApplicationRecord
   validates :name, presence: true
+  validates :video_link, url: { no_local: true, schemes: ['https'] }, length: { in: 10..100 }, allow_blank: true
```

Comment:
Do not use hardcoded numbers or other values like that. Everyone will immediately wonder why 10-100 not for example 20-200. Code should be written in a way that the intent is clear
```suggestion
  URL_MIN_LENGTH = 10
  URL_MAX_LENGTH = 100
  
  validates :video_link, url: { no_local: true, schemes: ['https'] }, length: { in: URL_MIN_LENGTH..URL_MAX_LENGTH }, allow_blank: true
```
When you do that you'll see that URLs cannot be capped at 100 characters. They are 2048 max (https://www.google.com/search?q=url+max+length&oq=url+max+le&aqs=chrome.0.0i512j69i57j0i512l4j0i22i30l4.1280j0j7&sourceid=chrome&ie=UTF-8).
Also why do we check for URL length? Is URL validator not enough?